ee/tabby-db/migrations/0014_password-reset.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE password_reset(
+    id INTEGER PRIMARY KEY NOT NULL,
+    code VARCHAR(36) NOT NULL,
```

Comment:
still need a field to reference user table? e.g `user_id: INTEGER`

Reply:
The `id` in here refers to the user's ID, not a row ID. I can update the column name so that it's more clear.

Reply:
let's have both `id` and `user_id` to the table.

Always have an `id` as primary key for any table.

Reply:
Curious to know why, if it will never be used? Do we want to allow the same user to have multiple active password reset codes at once (potentially confusing)? And if not, should the primary key then be `(id, user_id)`?

Reply:
https://stackoverflow.com/questions/1207983/in-general-should-every-table-in-a-database-have-an-identity-field-to-use-as-a

The primary key should be id. you can add unique constraint to `user_id`.